pom.xml

```diff
@@ -182,10 +182,29 @@ along with this program.  If not, see <http://www.gnu.org/licenses />.
                     </exclusion>
                 </exclusions>
             </dependency>
+            <dependency>
+                <groupId>com.codeablereason.restcompress.provider</groupId>
+                <artifactId>rest-compress-lib</artifactId>
+                <version>0.6</version>
+            </dependency>
         </dependencies>
     </dependencyManagement>
     <dependencies>
         <dependency>
+            <groupId>com.codeablereason.restcompress.provider</groupId>
```

Comment:
I will remove, I thought that when I made but I also thought it would be nice to it be available to all front end projects at once as they may have new rest interfaces could emerge into the other projects of this repository in the future. I 1ill update this PR in a moment